README.md

```diff
@@ -1,36 +1,270 @@
+```ruby
+config.environment :lan do |lan|
+  ip = Socket.ip_address_list.detect { |intf| intf.ipv4? && !intf.ipv4_loopback? && !intf.ipv4_multicast? }.ip_address
+  lan.setting apiEndpoint: "http://#{ip}:3000" # Or however you specify your server's location.
+  # lan.javascripts "resources/vendor/javascripts/some-library-debug-all.js"
+end
+
+config.environment :dev do |dev|
+  dev.setting apiEndpoint: "http://dev.example.com"
+  # dev.javascripts "lib/some-library-minified.js"
+end
+
+config.environment :production do |production|
+  production.setting apiEndpoint: "http://example.com"
+  # production.javascripts "lib/some-library-minified.js"
+  # production.minify :uglifier # or :yui
+end
+```
+
+## Usage
+
+* `$ hi5 deploy <platform> -e <environment>`
+
+This will build your application for a specific platform with the settings for the given environment to the destination directory specified in `config/high_five.rb`.
+
+* `$ hi5 <android|ios>:set_icon path/to/icon.png`
+
+By default, high\_five uses ChunkyPNG to resize the icon, but this almost always bad icons.
+We highly recommend adding the [rmagick](https://github.com/rmagick/rmagick) gem (which depends on imagemagick) to your Gemfile along with high\_five.
+High Five will detect that you have rmagick installed and will use it to generate good icons.
+
+* `$ hi5 <android|ios>:set_version -v "1.0" -b 200`
+
+This sets the publicly visible "version string" and the build number. They are in either `info.plist` or `AndroidManifest.xml` for iOS and Android, respectively.
+
+* `$ hi5 dist <platform>`
+
+Compiles the native application to an IPA or APK. This is optional, and instead you may use Xcode, Android Studio, gradle, or any other similar tool.
+
+If you pass it the `--install` flag, it will attempt to install to a connected device.
+
+* There's probably more high five commands that should be documented. If you find any not listed here, open an issue or submit a PR.
+
+### Development Usage
+
+#### Build and deploy for browser
+
+`hi5 deploy web`
+
+Then open index-debug.html inside of google chrome with web security disabled (`google-chrome --disable-web-security`)
+
+#### Build and deploy for device
+
+Make sure your device and computer are on the same wifi network (subnet).
+
+Make sure your server accepts connects from devices on the same subnet.
+
+e.g. For rails, start your server like this:
+
+`rails server -b 0.0.0.0`
+
+##### Android
+
+According to Brett, only the _first_ time you put a build on your device, you need to do this (replace the keystore, alias, password, and alias password with the correct values for your app):
+
+`hi5 deploy android --environment lan && hi5 dist android --environment lan --install --ant -o "Foobar-android--adhoc" -e --ant-flags="-Dkey.store=foobar.keystore -Dkey.alias=foobar -Dkey.store.password=foobar -Dkey.alias.password=foobar"`
```

Comment:
So i think you should remove --ant and the -o from this because its not necessary.